src/components/editTestCase/qiCore/LeftPanel/ElementsTab/Demographics/DemographicsSection.tsx

```diff
@@ -24,12 +27,18 @@ import {
 } from "./DemographicUtils";
 import "./DemographicsSection.scss";
 import _ from "lodash";
-import { FormControl, MenuItem as MuiMenuItem } from "@mui/material";
+import {
+  FormControl,
+  InputAdornment,
+  InputLabel,
+  MenuItem as MuiMenuItem,
+} from "@mui/material";
```

Comment:
InputAdornment, InputLabel imports can be removed as they are not being used anywhere